server/src/main/java/io/littlehorse/server/streamsimpl/coreprocessors/KafkaStreamsLHDAOImpl.java

```diff
@@ -976,6 +987,7 @@ private void clearThingsToWrite() {
         wfRunPuts.clear();
         wfSpecPuts.clear();
         taskDefPuts.clear();
+        taskRunPuts.clear();
```

Comment:
for now this is ok, but if we want to prevent for happening again I recommend to use a **queue** or a similar collection where all objects (variablePuts, nodeRunPuts, .....) , it is not a cache is more like a queue of stuffs to be committed.

Reply:
Yeah, we probably could have just used one thing for all of the types.